### DISCUSSION TITLE
Panon is not working

### DISCUSSION THREAD

**Discussion by cyra-tech** [community/community#51395] [Other Feature Feedback, Questions, & Ideas]
Question

HI
I installed Panon 0-4-6 and added its widget to desktop, but its not work and it's just a white rectangle on the screen. I installed all panon Dependencies and requirements but none solved the problem.
I tried all the suggested solutions on github but it didn't work.

https://github.com/rbn42/panon

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by cyra-tech** [reply]
Thank you for replying.  But my problem is not how to use Panon.  The problem is that it doesn't work.I installed it correctly and installed all its prerequisites, but when I add the widget to the desktop, I just get a white rectangle.
